src/elements/content-picker/stories/ContentPicker.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from '@storybook/blocks';
+
+import ContentPickerStories from './ContentPicker.stories';
+
+<Meta of={ContentPickerStories} />
```

Comment:
Needed this so that it places the doc in the right location

Reply:
thanks for pointing this out! I went and read about how this works here:
https://storybook.js.org/docs/api/doc-block-meta